install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1699,13 +1704,204 @@ dashboards:
 
 # -- Configure Istio proxy options.
 proxy:
+
   prometheus:
+    # -- Deprecated in favor of envoy.prometheus.enabled
     enabled: true
-    port: "9964"
+    # -- Deprecated in favor of envoy.prometheus.port
+    port: ~
   # -- Regular expression matching compatible Istio sidecar istio-proxy
   # container image names
   sidecarImageRegex: "cilium/istio_proxy"
 
+# -- Configure Cilium Envoy options.
+envoy:
+  # Enable Envoy Proxy in standalone DaemonSet.
+  enabled: false
+
+  # Location of unix sockets for communication between Agent & Envoy.
+  socketDir:
+    # Socket directory within the envoy & agent containers
+    containers: "/var/run/cilium/envoy/sockets"
```

Comment:
Why should this be configurable? Would this ever be changed?

Reply:
@squeed within the agent - we're building on top of the `runDir` when creating the sockets dir - which unfortunately is configurable. 

e.g. if someone is changing this he should have an option to change it here as well to reflect it in the external DaemonSet.

Reply:
This path should be a fixed path on the containter side. And on the host side this should be relative to the agent's `runDir`. This way no new helm values are needed for these paths.